sbin/rss-daemon.sh

```diff
@@ -181,6 +181,27 @@ case $option in
     fi
     ;;
 
+  (restart)
+
+    if [ -f $pid ]; then
+      TARGET_ID="$(cat "$pid")"
+      if [[ $(ps -p "$TARGET_ID" -o comm=) =~ "java" ]] || [[ $(ps -p "$TARGET_ID" -o comm=) =~ "jboot" ]]; then
+        echo "stopping $command"
+        kill "$TARGET_ID" && rm -f "$pid"
+        while [[ $(ps -p "$TARGET_ID" -o comm=) != "" ]];
+        do
+          echo "waiting worker graceful shutdown, sleep 1s"
+          sleep 1s
```

Comment:
Shall we add timeout here?

Reply:
> Shall we add timeout here?

I think we have a timeout in graceful shutdown, so don't need to do this here?

Reply:
In case something wrong with graceful shutdown, then restart script will hang forever

Reply:
> 

DOne